src/GlobalStyle.js

```diff
@@ -42,6 +43,7 @@ const GlobalStyle = createGlobalStyle`
       'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
       sans-serif;
     font-size: 1rem;
+
```

Comment:
Have look to the empty rows, line 3 is too much I think, 46 as well and between 47 and 48 😊